frontend/src/components/Form.jsx

```diff
@@ -3,22 +3,17 @@ import PropTypes from "prop-types";
 import _ from "lodash";
 import Modal from "./Modal";
 
-function Form({ fields, title, callback, isOpen, toggleModal }) {
+function Form({ fields, title, callback, isOpen, toggleModal, language }) {
   const [fieldsState, setFields] = useState(fields);
-  debugger;
+
   function onSave() {
-    debugger;
     let editedFields = _.difference(fieldsState, fields);
-    debugger;
     let editedByKey = _.groupBy(editedFields, "key");
-    debugger;
     editedByKey = _.transform(editedByKey, function (res, field, key) {
-      debugger;
       res[key] = _.head(field).value;
     });
-    debugger;
+    editedByKey.language = language;
     if (title === "dish") {
-      debugger;
       let form = new FormData();
```

Comment:
This logic should be outside since it's request/api related - here you should just call the callback with the params object